### PR TITLE
Increase the code coverage of services #4506

### PR DESCRIPTION
**Pull request by OlegVat**
Oleh Vatuliak 
#4506

## Summary of issue

Improved code coverage

## Summary of change

Added unit tests in the following classes:

1) CourierDtoMapper
    service/src/main/java/greencity/mapping/courier/CourierDtoMapper.java

2) EncryptionUtil class all methods
    service-api/src/main/java/greencity/util/EncryptionUtil.java

3) LocationToLocationsDtosMapper
    service/src/main/java/greencity/mapping/location/LocationToLocationsDtosMapper.java

### CODE REVIEW

**Review by innayashna — APPROVED** [review]

**Review by VadymMakitra — APPROVED** [review]

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            .rrn("IADG/0000000C00000000")
+            .sender_account("sender")
+            .sender_cell_phone("+4930901820")
+            .sender_email("sender@mail.com")
+            .settlement_currency("USD")
+            .settlement_date("03/08/22")
+            .tran_type("purchase")
+            .verification_status("ACTIVE")
+            .parent_order_id(1)
+            .build();
```
Comment (danylohlynsky) [inline service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Can you move this object creation to ModelUtils?

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    public void formRequestSignature() {
+        PaymentRequestDto paymentRequestDto = PaymentRequestDto.builder()
+            .orderId("1")
+            .merchantId(2)
+            .orderDescription("")
+            .currency("USD")
+            .amount(2)
+            .signature("")
+            .responseUrl("responseUrl")
+            .build();
```
Comment (danylohlynsky) [inline service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Same

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            .version(3)
+            .action("pay")
+            .amount(2)
+            .currency("USD")
+            .description("description")
+            .orderId("233")
+            .language("eng")
+            .paytypes("card")
+            .resultUrl("resultUrl")
+            .build();
```
Comment (danylohlynsky) [inline service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Same and also in all other tests

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java (hunk 1 above)
Comment (OlegVat) [reply service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Done

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java (hunk 2 above)
Comment (OlegVat) [reply service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Done

File: service-api/src/test/java/greencity/util/EncryptionUtilTest.java (hunk 3 above)
Comment (OlegVat) [reply service-api/src/test/java/greencity/util/EncryptionUtilTest.java]: Done

**Review by danylohlynsky — APPROVED** [review]

**Pull request merged**